.github/workflows/python-poetry-release.yaml

```diff
@@ -79,14 +80,16 @@ jobs:
           poetry-version: ${{ inputs.poetry-version }}
           working-directory: ${{ inputs.working-directory }}
 
-      - name: Release to PyPI
-        uses: bakdata/ci-templates/actions/python-poetry-release@v1.18.2
+      - name: Create changelog
+        id: build_changelog
+        uses: bakdata/ci-templates/actions/changelog-generate@fix/bug-changelog
```

Comment:
TODO

Reply:
This is already fix. So I will close it :)